CHANGELOG.md

```diff
@@ -270,6 +270,8 @@
 * README: Recommend running buildenvs over sourcing them on Linux [#13071](https://github.com/mixxxdj/mixxx/pull/13071)
 * FindSndFile: Link mpg123 in static builds [#13087](https://github.com/mixxxdj/mixxx/pull/13087)
 * macOS packaging: Enable app sandbox in ad-hoc-packaged (i.e. non-notarized) bundles too [#12101](https://github.com/mixxxdj/mixxx/pull/12101)
+* Drop support for macOS versions earlier than 11, Windows 7 and 8.1 and Ubuntu 20.04 "Focal Fossa" and older.
```

Comment:
Windows 8 support and support for Windows 10 builds before 1809 is also dropped.
But I think, that of the mentioned versions, only Windows 7 has a significiant user base nowadays.

Reply:
done.